sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -2114,6 +2114,20 @@ val SHUFFLE_COMPRESSION_LZ4_CHUNK_SIZE = conf("spark.rapids.shuffle.compression.
     .booleanConf
     .createOptional
 
+  val TEST_GET_JSON_OBJECT = conf("spark.rapids.sql.test.get_json_object")
+      .doc("Only for tests: verify for get_json_object. If enabled, dump the data that " +
+          "causes differences between Spark and GPU.")
+      .internal()
+      .booleanConf
+      .createWithDefault(false)
+
+  val TEST_GET_JSON_OBJECT_SAVE_PREFIX = conf("spark.rapids.sql.test.get_json_object.save.prefix")
```

Comment:
nit: ```spark.rapids.sql.test.get_json_object.dump.file.dir``` better name?

Reply:
Now one day generates a file.
By default, csv file name format is: /tmp/get-json-object_diffs_yyyyMMdd.csv